frontend/app/controllers/auth_controller.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = function(app) {
+  app.controller('AuthController', ['$http', '$location', function($http, $location) {
+
+    this.user = {};
```

Comment:
Not sure what your plan is with this data, but if you plan on storing/reusing the user data then it might be better to create a service for this (e.g. `AuthenticationService`) since that should be a singleton that lives during the lifespan of the app.